doc/playground.lua

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/tarantool
```

Comment:
It will make your script more portable:
```suggestion
#!/usr/bin/env tarantool
```

Reply:
Ouch, yep, I usually do, but forgot now. Fixed.